typings/numba.pyi

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: PYI033
```

Comment:
Why is this needed? Is this a bug in ruff? This file has comments, such as the typeshed-recommended `# incomplete`, but no type comments

Reply:
The second line has a PYI033 problem.

Reply:
That should be a bug in ruff :)

If possible (not sure whether it interferes with pyright), can you move the ruff comment after the pyright comment: `# pyright: reportIncompleteStub = false # ruff: noqa: PYI033`

Reply:
Then Pylance raise an syntax error:
```
Pyright comment directive must be followed by "=" and a value of true, false, error, warning, information, or none
```

Reply:
Then leave it as is.

Opened https://github.com/charliermarsh/ruff/issues/4151

Reply:
a new ruff version with the fix released :)